source.go

```diff
@@ -51,6 +51,13 @@ func NewSchemaSource(uri string) (SchemaSource, error) {
 		return NewReaderSource(os.Stdin), nil
 	}
 
+	if strings.HasPrefix(uri, "mysql://") {
+		// Treat the argument as a DSN for mysql.
+		// DSN is everything after "mysql://", so let's be lazy
+		// and use everything after the second slash
+		return NewMySQLSource(uri[8:]), nil
+	}
```

Comment:
We call mysql.ParseDSN on `NewMySQLSource` so, not calling url.Parse for mysql protocol is not problem.